recipes/prefect-ray/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "prefect-ray" %}
+{% set version = "0.2.0.post2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 763effd3806e493a901c7542b8f3e25a92806222669b33cdb335c70a53050822
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  # pickle5 only available for python 3.7
+  skip: true  # [py<37,osx]
```

Comment:
conda-forge will soon [drop Python 3.7](https://conda-forge.org/docs/user/announcements.html#dropping-python-3-7). If you are fine with this package only being available for >= 3.8, you could make this recipe `noarch: python` and put that min version on Python itself.